go.mod

```diff
@@ -21,4 +22,5 @@ require (
 	golang.org/x/sync v0.0.0-20201207232520-09787c993a3a
 	google.golang.org/grpc v1.35.0
 	google.golang.org/protobuf v1.25.0
+	sigs.k8s.io/controller-runtime v0.8.1
```

Comment:
Do you know why this module is listed?

Reply:
It because my wrong import...
I'll replace it with `go.uber.org/zap`